src/components/MentionSuggestions.tsx

```diff
@@ -70,6 +72,7 @@ function MentionSuggestions({prefix, mentions, highlightedMentionIndex = 0, onSe
                             size={isIcon ? CONST.AVATAR_SIZE.MENTION_ICON : CONST.AVATAR_SIZE.SMALLER}
                             name={item.icons[0].name}
                             type={item.icons[0].type}
+                            fill={theme.success}
```

Comment:
This line addition created problem. More details here: https://github.com/Expensify/App/issues/33657